### PR TITLE
Prepare multi-arch support

### PR DESCRIPTION
**Pull request by steveworley**
Prepare multi-arch support for better compatibility with Apple M1 (arm64) processors.

- Test the publish stage

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,50 +3,76 @@ version: 2
...
       - setup_remote_docker:
           version: 20.10.7
       - run:
-          name: Bring the containers up.
-          command: docker-compose up -d
+          name: Prepare the docker context
+          command: |
+            docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+            docker context create dpc
+            docker buildx create --platform linux/amd64,linux/arm64 dpc --name dpc --use
```
Comment (christopher-hopper) [inline .circleci/config.yml:21]: My Apple M1  says its detected platform is `linux/arm/v8` so maybe need to build for that too? like:

```
docker buildx create --platform linux/amd64,linux/arm64,linux/arm/v8 
```

**Comment by christopher-hopper** [comment]
@steveworley 
It looks like changes from this branch were merged. Assume this PR can be closed?

**Comment by christopher-hopper** [comment]
@steveworley @GROwen 
I'm going to continue work in this branch to further progress multi-arch support.

**Comment by christopher-hopper** [comment]
MariaDB build works and new arm64 image is being built from the arm64 compatible base. You can see that it is slightly smaller than the amd64 image, which is as expected. 

Screen Shot 2022-05-03 at 4 04 55 pm


The image however has no customisations from the base uselagoon/mariadb-10.4-drupal image. As such, you could really stop building and publishing this image altogether.

**Comment by GROwen** [comment]
Closing in favour of https://github.com/dpc-sdp/bay/pull/108

**Pull request closed without merging**